src/ui/public/notify/notifier.js

```diff
@@ -310,48 +320,69 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  * @param  {Function} cb
  */
-Notifier.prototype.error = function (err, cb) {
-  return add({
+Notifier.prototype.error = function (err, opts, cb) {
+  const overrideable = ['lifetime', 'icon'];
```

Comment:
Does it make sense to extract `overrideable` into a higher scope as it is used by multiple methods?

Also, I'd suggest naming this a bit more descriptively, like `overrideableOptions` or `overrideableOpts`.